Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC
+
+RUN apt-get update && apt-get install -y \
+    apt-transport-https \
+    curl \
+    git \
+    locales \
+    mysql-client \
+    nginx \
+    software-properties-common \
+    unzip \
+    tzdata \
+    vim
+
+# System-level PHP setup
+RUN add-apt-repository ppa:ondrej/php -y
+RUN apt-get install -y \
+    php7.4 php7.4-curl php7.4-fpm php7.4-gd php7.4-mbstring php7.4-mysql php7.4-xml
+RUN mkdir /run/php # For the php-fpm socket file
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN sed -i -e '/catch_workers_output =/s/.*/catch_workers_output = yes/' /etc/php/7.4/fpm/pool.d/www.conf
+
+# nginx setup
+# note that these symlinks require the repo be mounted @ /var/www/openflights
+RUN rm /etc/nginx/sites-enabled/default
+RUN ln -s /var/www/openflights/nginx/openflights-dev /etc/nginx/sites-available/
+RUN ln -s /var/www/openflights/nginx/openflights-dev /etc/nginx/sites-enabled/
```

Comment:
Isn't it more usual to chain the `sites-enabled/foo` to symlink to `sites-available/foo`, even if that is a symlink itself?

Reply:
/shrug

We don't even need the `sites-available` one.